### ISSUE TITLE
Online access to AMD hardware

### ISSUE THREAD

**Issue by deepankarsharma** [RadeonOpenCompute/ROCm#153]
I am looking to port some performance sensitive applications to using HIP but am having a hard time buying AMD hardware at sensible prices given the mining boom going on. I was wondering if there are a couple of machines with remote access where developers can log in to get familiar with the stack and use for porting efforts?

**Comment by gstoner** [comment]
We currently do not have Online access to hardware running ROCm.  It is some thing we are actively working on

**Comment by nevion** [comment]
just wanted to say I've also requested this a few times in the past with @johnbridgman, ended up going nowhere though (not to John's fault, the problems were in the details and network security IIRC)

Would make it so much easier to show where the hardware and compiler and library stacks shines... and track that publicly each release.  Such as with the arrayfire-pts plugin I wrote.  The difficulty isn't the software for me, it's hardware access...  Particularly in the enterprise level cards.  It's so difficult I was thinking of trying the patreon thing to fund the hobby/hardware acquisition necessary for testing these things.  Businesses make decisions on these on such results that show what the hardware+software can do when pushed to the limit in specific microbenchmarks, esp those scientific computing workloads care about. At the moment the closest resource to this is the phoronix test suite + phoronix.
